templates/todo/tag_list.html

```diff
@@ -0,0 +1,40 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <h1>Tags</h1>
+    <button style="float: right; margin-right: 25px; bottom: 50px; position:relative;" class="btn btn-primary">
+        <a role="button" style="color: white"
+           href="{% url 'todo:tag-create'%}">Add</a>
+    </button>
+
+    {% if tag_list %}
+        <table class="table">
+            <tr>
+                <th>Name</th>
+                <th>Update</th>
+                <th>Delete</th>
+            </tr>
+            {% for tag in tag_list %}
+                <tr>
+                    <td>{{ tag.name }}</td>
+                    <td>
+                        <button style="background: gray" class="btn"><a role="button" style="color: white"
```

Comment:
Here and in line 26 the code is almost the same. You can add here class btn-grey and move style="background: grey" to styles.css. Also style="color: white" you can move to styles.css too, assign class btn-grey to tag a.
Add styles to the css file:

.btn-grey {
    background-color: grey;
}

.button-link{
    color: white
}

```suggestion
                        <button class="btn btn-grey"><a class="button-link" role="button" 
```